packages/react-liveness/src/components/FaceLivenessDetector/service/utils/liveness.ts

```diff
@@ -688,3 +689,14 @@ export async function isFaceDistanceBelowThreshold({
 
   return { isDistanceBelowThreshold, error };
 }
+
+export const resolveVideoDimensions = (
+  params: VideoAssociatedParams
+): Pick<MediaTrackSettings, 'height' | 'width'> => {
+  const mediaSettings = params.videoMediaStream?.getTracks()[0].getSettings();
+  const { videoHeight, videoWidth } = params.videoEl ?? {};
+  return {
+    height: mediaSettings?.height ?? videoHeight,
+    width: mediaSettings?.width ?? videoWidth,
```

Comment:
Nit: rather than return `height` and `width`, can provide the return values as `videoHeight` and `videoWidth` and handle the `toString` calls here so the consumer can simply spread the result:
```
       responseStream = getResponseStream({
          ...resolveVideoDimensions(
             context.videoAssociatedParams!
          ),
          requestStream,
          sessionId: context.componentProps!.sessionId,
        });
```

Reply:
Will address in follow up PR.